code/obj/vortex.dm

```diff
@@ -52,45 +67,58 @@
-			/obj/critter/gunbot/heavy,
-			/obj/machinery/bot/medbot/terrifying,
-			/obj/machinery/bot/medbot/terrifying)
-			if(prob(3))
-				horror_path = pick(/obj/critter/gunbot/drone/buzzdrone,/obj/critter/gunbot/drone/buzzdrone, /obj/critter/aberration)
-			if (was_eaten && prob(15))
-				horror_path = /obj/critter/blobman/meaty_martha
+		//spatial interdictor: when something would exit a vortex, it doesn't
+		//consumes 500 units of charge per inbound thing interdicted
+		var/interdicted = FALSE
+		for_by_tcl(IX, /obj/machinery/interdictor)
+			if (IX.expend_interdict(500,src))
+				interdicted = TRUE
+				break
+		if(!interdicted)
+			if(derelict_mode)
+				horror_path = pick(/obj/critter/shade,
+				/obj/critter/shade,
+				/obj/critter/shade,
+				/obj/critter/shade,
+				/obj/critter/shade,
+				/obj/critter/crunched,
+				/obj/critter/crunched,
+				/obj/critter/crunched,
+				/obj/critter/bloodling,
+				/obj/critter/ancient_thing,
+				/obj/critter/ancient_thing,
+				/obj/critter/ancient_repairbot/grumpy,
+				/obj/critter/ancient_repairbot/grumpy,
+				/obj/critter/ancient_repairbot/security,
+				/obj/critter/ancient_repairbot/security,
+				/obj/critter/gunbot/heavy,
+				/obj/machinery/bot/medbot/terrifying,
+				/obj/machinery/bot/medbot/terrifying)
+				if(prob(3))
+					horror_path = pick(/obj/critter/gunbot/drone/buzzdrone,/obj/critter/gunbot/drone/buzzdrone, /obj/critter/aberration)
+				if (was_eaten && prob(15))
+					horror_path = /obj/critter/blobman/meaty_martha
+			else
+				horror_path = pick(/obj/critter/killertomato,
+				/obj/critter/spore,
+				/obj/critter/spacerattlesnake,
+				/obj/critter/martian/warrior,
+				/obj/machinery/bot/firebot/emagged,
+				/obj/machinery/bot/secbot/emagged,
+				/obj/machinery/bot/medbot/mysterious/emagged,
+				/obj/machinery/bot/cleanbot/emagged,
+				/obj/critter/wasp/angry,
+				/obj/critter/spacescorpion,
+				/obj/critter/mimic,
+				/obj/critter/fermid,
+				/obj/critter/bear)
+			var/obj/horror = new horror_path(src.loc)
+			src.visible_message("<span class='alert'><b>[horror] emerges from the [src]!</b></span>","<span class='alert'>You hear a sharp buzzing noise.</span>")
 		else
-			horror_path = pick(/obj/critter/killertomato,
-			/obj/critter/spore,
-			/obj/critter/spacerattlesnake,
-			/obj/critter/martian/warrior,
-			/obj/machinery/bot/firebot/emagged,
-			/obj/machinery/bot/secbot/emagged,
-			/obj/machinery/bot/medbot/mysterious/emagged,
-			/obj/machinery/bot/cleanbot/emagged,
-			/obj/critter/wasp/angry,
-			/obj/critter/spacescorpion,
-			/obj/critter/mimic,
-			/obj/critter/fermid,
-			/obj/critter/bear)
-		var/obj/horror = new horror_path(src.loc)
-		src.visible_message("<span class='alert'><b>[horror] emerges from the [src]!</b></span>","<span class='alert'>You hear a sharp buzzing noise.</span>")
+			SPAWN(rand(2,20)) //desynchronize the visual/audible indication of interdiction in case of large batches of simultaneous vortexes
```

Comment:
```suggestion
			SPAWN(rand(0.2 SECONDS, 2 SECONDS)) //desynchronize the visual/audible indication of interdiction in case of large batches of simultaneous vortexes
```